loleaflet/src/control/Control.JSDialogBuilder.js

```diff
@@ -1863,6 +1863,21 @@ L.Control.JSDialogBuilder = L.Control.extend({
 		builder._createComment(container, data, true);
 		if (data.children.length > 0)
 		{
+			var numberOfReplies = data.children.length - 1;
+			if (numberOfReplies > 0)
+			{
+				var replyNode = L.DomUtil.create('div','loleaflet-annotation-content',$(container).find('.loleaflet-annotation-content-wrapper')[0]);
+				var replyContent = L.DomUtil.create('div', '', replyNode);
```

Comment:
Do we really need this node? can't we just put text inside `replyNode` directly?